tests/beaker_tests/cisco_bgp_neighbor_af/test_bgpneighboraf.rb

```diff
@@ -539,154 +603,170 @@ def build_manifest_bgp_nbr_af(tests, id)
 
 # Wrapper for bgp_nbr_af specific settings prior to calling the
 # common test_harness.
-def test_harness_bgp_nbr_af(tests, id)
+def test_harness_bgp_nbr_af(tests, id, platform)
   af = af_title_pattern_munge(tests, id, 'bgp_neighbor_af')
   logger.info("\n--------\nTest Case Address-Family ID: #{af}")
 
-  # Set up remote-as if necessary
-  bgp_nbr_remote_as(agent, tests[id][:remote_as]) if tests[id][:remote_as]
-
   tests[id][:ensure] = :present if tests[id][:ensure].nil?
-  tests[id][:show_pattern] = af_pattern(tests, id, af)
   tests[id][:resource_cmd] = puppet_resource_cmd(af)
 
-  # Build the manifest for this test
-  build_manifest_bgp_nbr_af(tests, id)
+  if platform == 'ios_xr' && af[:vrf] != 'default'
```

Comment:
This looks like we aren't testing nbr af under non-default vrfs.  Is that intended?